councilmatic/settings_jurisdiction.py

```diff
@@ -233,6 +233,119 @@
     },
 ]
 
+BILL_STATUS_DESCRIPTIONS = [
+    {
+        'name': 'ADOPTED',
```

Comment:
With this structure, we duplicate the mapping of action to status: It is now here and in the `_status` method on the `LAMetroBill`. It seems right to store this mapping as a configuration variable. Can you propose a different structure for this dictionary that you could use to populate the About page and also to determine the corresponding status to a given action in the `_status` method?

Reply:
I'm not sure if this is what you mean, but I could remove the `search_term` key from each dictionary & add the `search_term` to the dictionary using the `_status` method in the view?

My only concern with this is that there are a few of these bill status definitions that aren't accounted for in the `_status` method. Those include all of the `under reconsideration` statuses:
* APPROVED UNDER RECONSIDERATION
* FAILED UNDER RECONSIDERATION
* ADOPTED UNDER RECONSIDERATION
* APPROVED AS AMENDED UNDER RECONSIDERATION
* ADOPTED AS AMENDED UNDER RECONSIDERATION

I thought these fit pretty clearly within the status facet buckets we currently have, so I added those `search_term`s.

Reply:
Let's talk out loud about this. Can you put 5 minutes on the calendar when you next plan to work on Metro?

Reply:
I will likely start working on metro after our checkin next week. can we talk during our meeting?

Reply:
Let's do it! I updated the event name to Sprint planning/pairing as a reminder.